addon/components/hyper-table-v2/column.ts

```diff
@@ -43,4 +72,29 @@ export default class HyperTableV2Column extends Component<HyperTableV2ColumnArgs
 
     return `hypertable__column--size-${size}`;
   }
+
+  @action
+  orderColumn(e: MouseEvent): void {
+    e.stopPropagation();
+
+    if (!this.args.column.definition.orderable) return;
+
+    this.args.handler.applyOrder(this.args.column, this.args.column.order?.direction === 'desc' ? 'asc' : 'desc');
+  }
+
+  @action
+  toggleFilteringComponent(e: MouseEvent): void {
+    e?.stopPropagation?.();
```

Comment:
```suggestion
    e.stopPropagation.();
```
Do you need all of the `?` here?

Reply:
Yep ! when this is triggered via the `on-click-outside`, the event isn't passed :(